Testreglar/251/App/app-2.5.1a.json

```diff
@@ -0,0 +1,127 @@
+{
+	"id": "app-2.5.1a",
+	"type": "app",
+	"kravTilSamsvar": "<p>All funksjonalitet som krever at brukeren benytter flerpunktgest eller stibasert gest, skal kunne betjenes med en enkelt pekerbevegelse</p>\r\n<ul>\r\n<li>Med mindre flerpunktgest eller stibasert gest er avgjørende</li>\r\n</ul>",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		}
+	],
+	"side": "2.1",
+	"element": "3.1",
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken appside tester du?",
+			"ht": "<p>Oppgi appside-ID.</p>",
+			"type": "tekst",
+			"kilde": [],
+			"label": "Appside:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Finnes det funksjonalitet som krever at brukeren benytter flerpunktsgest eller stibasert gest?",
+			"ht": "<p>Eksempler på flerpunktsgest eller stibasert gjest er:</p>\n<ul>\n<li>Flerpunktsgest: funksjonalitet for å zoome ved å knipe fingrene på berøringsskjerm, trykk med to eller tre fingre, tegne en \"Z\" på skjermen angrer siste handling.</li>\n<li>Stibasert gjest: sveiping i en bestemt retning eller tegning av en bestemt form, f.eks. bilde-karuseller, glide-brytere eller sveiping for å utføre handling.</li>\n</ul>\n<p><strong>Slik identifiserer du en funksjonalitet som krever at brukeren benytter flerpunktsgest eller stibasert gest:</strong></p>\n<ul>\n<li>Slå av skjermleser og andre andre tilgjengelighetsfunksjoner, hvis det er aktivert på enheten.</li>\n<li style=\"list-style-type: none;\">\n<ul>\n<li>Dette gjelder både hjelpemiddelteknologi og innstillinger i operativsystemet. </li>\n</ul>\n</li>\n<li>Åpne appsiden</li>\n<li>Gjør en visuell inspeksjon for å identifisere funksjonalitet der det kan være aktuelt at innholdet krever at brukeren trykker flere steder på skjermen samtidig eller beveger pekeren i bestemte mønster\n<ul>\n<li>Finn funksjonalitet og sjekk om innholdet bruker flerpunktsgest eller stibaserte gester </li>\n</ul>\n</li>\n</ul>\n<p><strong>Du skal ikke teste:</strong></p>\n<ul>\n<li>Frihåndsbevegelser der startpunkt til sluttpunkt ikke krever at brukeren er innom et bestemt mellompunkt på veien, det vil si at brukeren følger en bestemt retning eller et mønster \n<ul>\n<li>Dra og slipp-funksjonalitet</li>\n<li><strong>Merk:</strong> Dra- og slipp-funksjonalitet som krever at pekeren flyttes i en bestemt retning eller rekkefølge (opp, ned, høyre eller venstre) er omfattet</li>\n</ul>\n</li>\n<li>Bruk med tastatur. Dette testes i 2.1.1</li>\n</ul>",
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Appsiden har ikke funksjonalitet som krever flerpunkt eller stibasert gjest."
```

Comment:
samme som for testregel nett. Utfall bør samsvare med spørsmål. Appsiden har ikke funksjonalitet som krever flerpunktsgest eller stibasert gest (ikke gjest)